### PR TITLE
Fixbug: increase the stdout/stderr buffer of exec

### PR DESCRIPTION
**Pull request by barretlee**

### CODE REVIEW

**Comment by barretlee** [comment]
@joaomoreno, please accept the PR, and publish a new version of npm package, thanks.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vsce",
-  "version": "1.53.2",
+  "version": "1.53.3",
```
Comment (joaomoreno) [inline package.json]: Please do not increase the version.

**Comment by ahonn** [comment]
Please accept the PR !!

**Comment by joaomoreno** [comment]
Released 1.57.1

**Pull request merged**